citesphere/src/main/webapp/WEB-INF/views/auth/group/item.jsp

```diff
@@ -250,129 +257,222 @@
+			</div>
+		</div>
+		<!-- /.modal-content -->
+	</div>
+	<!-- /.modal-dialog -->
+</div>
+<!-- /.modal -->
 
 <script>
-//# sourceURL=modal.js
+	//# sourceURL=modal.js
 	$(function() {
-		$(".parse-ref-info").click(function(e) {
-			var clicked = e.target;
-			$("#ref-unparsed").text($(clicked).text());
-			$("#parsed-ref-authors").text($(clicked).attr('data-authors'));
-			$("#parsed-ref-title").text($(clicked).attr('data-title'));
-			$("#parsed-ref-year").text($(clicked).attr('data-year'));
-			var identifier = $(clicked).attr('data-identifier');
-			var identType = $(clicked).attr('data-identifier-type');
-			if (identType != undefined && identType != "") {
-				identifier = identifier + " (" + identType + ")";
-			}
-			$("#parsed-ref-ident").text(identifier);
-			$("#parsed-ref-first-page").text($(clicked).attr('data-first-page'));
-			$("#parsed-ref-last-page").text($(clicked).attr('data-last-page'));
-			$("#parsed-ref-volume").text($(clicked).attr('data-volume'));
-			$("#parsed-ref-source").text($(clicked).attr('data-source'));
-			$("#parsed-ref-publication-type").text($(clicked).attr('data-publication-type'));
-			$("#parsed-ref-reference-id").text($(clicked).attr('data-reference-id'));
-			$("#parsed-ref-reference-label").text($(clicked).attr('data-reference-label'));
-			$("#parsed-ref-citation-id").text($(clicked).attr('data-citation-id'));
-			$("#parsed-ref-ref-raw").text($(clicked).attr('data-ref-raw'));
-            
-			$("#parsed-ref-modal").modal();
-		});
-	})
+		$(".parse-ref-info").click(
+				function(e) {
+					var clicked = e.target;
+					$("#ref-unparsed").text($(clicked).text());
+					$("#parsed-ref-authors").text(
+							$(clicked).attr('data-authors'));
+					$("#parsed-ref-title").text($(clicked).attr('data-title'));
+					$("#parsed-ref-year").text($(clicked).attr('data-year'));
+					var identifier = $(clicked).attr('data-identifier');
+					var identType = $(clicked).attr('data-identifier-type');
+					if (identType != undefined && identType != "") {
+						identifier = identifier + " (" + identType + ")";
+					}
+					$("#parsed-ref-ident").text(identifier);
+					$("#parsed-ref-first-page").text(
+							$(clicked).attr('data-first-page'));
+					$("#parsed-ref-last-page").text(
+							$(clicked).attr('data-last-page'));
+					$("#parsed-ref-volume")
+							.text($(clicked).attr('data-volume'));
+					$("#parsed-ref-source")
+							.text($(clicked).attr('data-source'));
+					$("#parsed-ref-publication-type").text(
+							$(clicked).attr('data-publication-type'));
+					$("#parsed-ref-reference-id").text(
+							$(clicked).attr('data-reference-id'));
+					$("#parsed-ref-reference-label").text(
+							$(clicked).attr('data-reference-label'));
+					$("#parsed-ref-citation-id").text(
+							$(clicked).attr('data-citation-id'));
+					$("#parsed-ref-ref-raw").text(
+							$(clicked).attr('data-ref-raw'));
+
+					$("#parsed-ref-modal").modal();
+				});
+    
```

Comment:
why did you change the indentation?

Reply:
since they were coming too far in github , i manually added spaces instead of tabs